zipkin-ui/js/component_ui/traceSummary.js

```diff
@@ -117,7 +117,9 @@ export function getServiceName(span) {
 }
 
 function getSpanTimestamps(spans) {
-  return _(spans).flatMap((span) => getServiceNames(span).map((serviceName) => ({
+  return _(spans)
+  .filter(span => span.timestamp != null && span.duration != null)
```

Comment:
the only scenario I can think of for this is that a span with only binary-annotations was returned by a query. I'll update this commit to add the comment.